packages/@aws-cdk/aws-efs/lib/efs-file-system.ts

```diff
@@ -1,15 +1,16 @@
 import * as ec2 from '@aws-cdk/aws-ec2';
 import * as kms from '@aws-cdk/aws-kms';
-import {Construct, Resource, Tag} from '@aws-cdk/core';
-import {CfnFileSystem, CfnMountTarget} from './efs.generated';
+import { Construct, IResource, Resource, Size, Tag } from '@aws-cdk/core';
+import { CfnFileSystem, CfnMountTarget } from './efs.generated';
 
-// tslint:disable: max-line-length
+// tslint:disable:max-line-length
```

Comment:
Is this really needed?

Reply:
Yes. link 2 lines below exceeds our 150 line width.